src/x86_64/unwind_rule.rs

```diff
@@ -19,6 +19,107 @@ pub enum UnwindRuleX86_64 {
     },
     /// (sp, bp) = (bp + 16, *bp)
     UseFramePointer,
+    /// sp = sp + 8 * (offset + register count)
```

Comment:
Hmm, this summary doesn't make it obvious why offset and register count are stored separately.